README.md

```diff
@@ -33,6 +34,8 @@ Function that extracts local and domain parts of email address.
         - For example, `todd` in `todd@underdog.io`
     - domain `String` - Tailing portion of email address
         - For example, `underdog.io` in `todd@underdog.io`
+    - full `String` - Full email address
+        - For example, `todd@underdog.io`
```

Comment:
We should provide context of when we get a full email address. Can we update this line to:

```
For example, `todd@underdog.io` for `todd@underdog.io`
```